ArduCopter/mode_flip.cpp

```diff
@@ -91,8 +89,12 @@ bool ModeFlip::init(bool ignore_checks)
 // should be called at 100hz or more
 void ModeFlip::run()
 {
+    // determine flip rotation rate
+    float flip_rate_cdps = g2.flip_rate_dps.get() * 100.0;
+    float flip_time_out_ms = 1000.0f * (36000.0f / MAX(flip_rate_cdps, 6000.0f) + 1.0);
```

Comment:
```suggestion
    const float flip_time_out_ms = 1000.0f * (36000.0f / MAX(flip_rate_cdps, 6000.0f));
```
You don't need the +1 because the max guarantees none zero.

Also could be const ;)

Reply:
The +1 was there to give extra time for the aircraft to achieve that rotational rate.  Think of it a fudge factor. The original timeout was at least a second longer than the rotational rate divided into 360.

Reply:
Ah, I see, a comment to that effect would be good. Would a multiplier be better than a fixed offset? At 400 deg/s the + 1 is ~ 2.1 times.